packages/core-plugin-v1/src/index.ts

```diff
@@ -58,3 +58,28 @@ export * from './runtime.ts';
 // - database
 // - knowledge / memory
 // - relationships
+
+// Export main runtime compatibility class
+export { CompatAgentRuntime } from './runtime';
```

Comment:
we're exporting runtime class as runtime so plugins written in v1 can talk to this core wrapper in v1 api and this talks to the v2 core